rotkehlchen/serialization/deserialize.py

```diff
@@ -205,6 +206,20 @@ def deserialize_timestamp_from_intms(time: int) -> Timestamp:
     return Timestamp(int(time / 1000))
 
 
+def deserialize_timestamp_ms(time: int) -> TimestampMS:
+    """Deserializes a timestamp in milliseconds from an integer.
+
+    May raise:
+    - DeserializationError
```

Comment:
This function is doing nothing. The linter would warn you that you are passing something that is not int to TimestampMS. If anything I believe the tipe of time should be `Any` and then the function might make sense
but then I would check if a conversion is possible. Basically the same as `deserialize_timestamp`.